cql3/statements/create_keyspace_statement.cc

```diff
@@ -115,7 +115,7 @@ future<std::tuple<::shared_ptr<cql_transport::event::schema_change>, std::vector
                 "To use CDC, LWT or counters, drop this keyspace and re-create it "
                 "without tablets by adding AND TABLETS = {'enabled': false} "
                 "to the CREATE KEYSPACE statement.");
-            if (ksm->initial_tablets().value()) {
+            if (ksm->initial_tablets().value_or(0)) {
```

Comment:
Why this change? It seems to have no effect.

Reply:
sorry, leftover from a previous version.
Will clean that up

Reply:
dropped in v7